test/extended/router/http2.go

```diff
@@ -378,3 +377,15 @@ func resolveHostAsAddress(oc *exutil.CLI, interval, timeout time.Duration, host,
 
 	return result, nil
 }
+
+// platformHasHTTP2LoadBalancerService returns true where the default
+// router is exposed by a load balancer service and can support http/2
+// clients.
+func platformHasHTTP2LoadBalancerService(platformType configv1.PlatformType) bool {
+	switch platformType {
+	case configv1.AWSPlatformType, configv1.AzurePlatformType, configv1.GCPPlatformType:
```

Comment:
This means we'll never test any new platforms unless we remember to add them here.  Isn't it better to be specific for the platforms we know will not need the test, to prevent silently passing new platforms that we should have tested?

Reply:
> This means we'll never test any new platforms unless we remember to add them here. Isn't it better to be specific for the platforms we know will not need the test, to prevent silently passing new platforms that we should have tested?

I've raised https://bugzilla.redhat.com/show_bug.cgi?id=1949154 to cover this. Pragmatically I'm just trying to avoid CI failures at the moment.

Reply:
The only platform that supports Load Balancer services at the moment is IBM cloud, right?